cirq/optimizers/merge_single_qubit_gates.py

```diff
@@ -121,3 +121,25 @@ def synth(qubit: 'cirq.Qid', matrix: np.ndarray) -> List[ops.Operation]:
         return [gate(qubit) for gate in out_gates]
 
     MergeSingleQubitGates(synthesizer=synth).optimize_circuit(circuit)
+
+
+def merge_single_qubit_gates_into_phxz(
+        circuit: circuits.Circuit,
+        atol: float = 1e-8,
+) -> None:
+    """Canonicalizes runs of single-qubit rotations in a circuit.
+
+    Specifically, any run of non-parameterized circuits will be replaced by an
```

Comment:
Would "any run of non-parameterized single-qubit operations" be more precise?

Reply:
That's definitely better; fixed.